code/DDSCodeTester.cpp

```diff
@@ -565,7 +565,9 @@ void dds_domain_examples()
 
         // Enable statistics DataWriter
         if (statistics_participant->enable_statistics_datawriter(eprosima::fastdds::statistics::GAP_COUNT_TOPIC,
-                eprosima::fastdds::statistics::dds::STATISTICS_DATAWRITER_QOS) != ReturnCode_t::RETCODE_OK);
+                eprosima::fastdds::statistics::dds::STATISTICS_DATAWRITER_QOS) != ReturnCode_t::RETCODE_OK)
+        {
+        }
```

Comment:
This file was uncrustified already in #300. And specifically these additional brackets were removed. In case that this PR is merged before the other, please fix it also here.